README.md

```
@@ -154,7 +154,112 @@ For Linux-specific builds (Flatpak, AppImage), see [guides/linux.md](guides/linu
+
+```bash
+cd services/convex
+```
+
+Copy the example environment file:
+
+```bash
+cp .env.example .env
+```
+
+Edit `.env` and fill in your values:
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)